Hymns/DetailHymnScreen.swift

```diff
@@ -3,6 +3,7 @@ import Resolver
 
 struct DetailHymnScreen: View {
     @State var favorited: Bool = false
+    @ObservedObject private var viewModel: HymnLyricsViewModel // Pass through to Hymn Lyrics View
```

Comment:
I think instead of observing the HymnLyricsViewModel, it'd make more to observe a DetailHymnScreenViewModel and have that expose the HymnLyricsViewModel for the HymnLyricsView to consume.

In the future, DetailHymnScreenViewModel would expose potentially 4 view models:
HymnLyricsViewModel
HymnChordsViewModel
HymnGuitarViewModel
HymnPianoViewModel

Reply:
Okay, I took a stab at restructuring this. Lmk what you think. I added a detailedHymnScreenVM to expose HymnLyricsViewModel